src/client.rs

```diff
@@ -812,10 +808,7 @@ impl Client {
     /// Screenshot](https://www.w3.org/TR/webdriver1/#dfn-take-element-screenshot) of the WebDriver
     /// standard.
     #[cfg_attr(docsrs, doc(alias = "Take Element Screenshot"))]
-    pub async fn screenshot_element(
-        &mut self,
-        element: Element,
-    ) -> Result<Vec<u8>, error::CmdError> {
+    pub async fn screenshot_element(&self, element: Element) -> Result<Vec<u8>, error::CmdError> {
```

Comment:
Seeing this, I wonder if this should be a method on `Element` instead? Incidental to this PR though.

Reply:
Probably. I'll move it (since we're already breaking everything 😅 )